include/af/defines.h

```diff
@@ -357,6 +357,15 @@ typedef enum {
 } af_image_format;
 #endif
 
+#if AF_API_VERSION >=34
+typedef enum {
+    M00 = 0,
```

Comment:
Changes:
MXY ->`AF_MOMENT_XY`.
af_moment -> af_moment_type

Also you missed adding `typedef af_moment_type af::momentType` in the `namespace af` later in the file.